### ISSUE TITLE
Fine-tunning BERT Video

### ISSUE THREAD

**Issue by mderosa001** [mderosa001/Capstone_Issues#18]
### Title
Fine-Tuning BERT for Text Classification (w/ Example Code)
### URL
https://www.youtube.com/watch?v=4QHg8Ix8WWQ
### Summary 
This youtube video breaks down fine-tuning, hyper parameterization, optimization, and loss functions. It is a hands on approach with code to get started fine-tunning your own BERT model. 
### Key Points 
fine-tunning
standard epochs
learning rates 
batch sizes
### Citation
@online{youtube,
        title = {Fine-Tuning BERT for Text Classification (w/ Example Code)},
        date = {2024},
        organization = {Youtube},
        author = {Shaw Talebi},
        url = {https://youtu.be/4QHg8Ix8WWQ?si=rfEjCkdrgd-KdTnh},
    }
